Models/SniperConfiguration.cs

```diff
@@ -33,5 +33,6 @@ public class SniperConfiguration
         public int BuyDelaySeconds { get; set; }
         public bool CheckRouterAddressInContract { get; set; }
         public bool OnlyBuyWhitelist { get; set; }
+        public double TrailingStopLoss { get; set; }
```

Comment:
Can we possibly make this a nullable, if it is null then the TrailingStopLoss logic should not happen

```suggestion
        public double? TrailingStopLoss { get; set; }
```

Reply:
Sure. No problem. Will fi that with the next commit.